src/library.js

```diff
@@ -453,7 +453,8 @@ LibraryManager.library = {
   clock__sig: 'i',
   clock: function() {
     if (_clock.start === undefined) _clock.start = Date.now();
-    return ((Date.now() - _clock.start) * ({{{ cDefine('CLOCKS_PER_SEC') }}} / 1000))|0;
+    const t = ((Date.now() - _clock.start) * ({{{ cDefine('CLOCKS_PER_SEC') }}} / 1000))|0;
+    return {{{ to64('t') }}};
   },
```

Comment:
`clock()` returns `clock_t` I think. Does musl define that differently in 64-bit? That surprises me a little as usually that stuff is fixed (say `ino_t` is always 64-bit).

Reply:
All I know is it ended up as an `i64` in the module.
Seems we have `typedef long clock_t;`

Reply:
Is that a typedef in musl code, or in emscripten code?

I ask because I wouldn't suggest changing musl code, but if it's in an emscripten header then maybe the simple change is to just make it always 32-bit.

Reply:
It looks like its in musl:

https://github.com/emscripten-core/emscripten/blob/4b9a0d01c7a8c7bb6cf3e00afee1e2d477ac4d32/system/lib/libc/musl/include/alltypes.h.in#L42

Reply:
We already have use of `long` all over the place, but ok, I'll try changing the type instead.

Reply:
I'm not opposed to `long` in principle - where we want wasm32 and wasm64 to diverge it sgtm - I'm just saying I think it isn't appropriate here specifically since we don't have a good reason to diverge the two targets. Do you disagree with that?

Reply:
No, it's all the same to me. I already changed the type and reverted the JS change.